workinggroups/networking.md

```diff
@@ -0,0 +1,53 @@
+# CNCF Networking WG Proposal
+
+## TOC Sponsor
+
+Ken Owens
+
+## Objective
+
+Explore cloud native networking technology and concepts around the container networking interface (CNI).
+
+## Goals and Expected Outcomes
```

Comment:
Does this WG still need to exist, or can it declare victory and dissolve?

Reply:
This WG exists today. Since this PR is about adding process and listing the current WGs should the discussion on dissolving the WG be moved to a meeting or mailing list?